openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -99,6 +99,14 @@ num_retries = 10
 [cinder]
 os_region_name = {{.Values.global.region}}
 cross_az_attach={{.Values.cross_az_attach}}
+auth_url = http://{{include "keystone_api_endpoint_host_internal" .}}:{{ .Values.global.keystone_api_port_internal | default "5000" }}/v3
+auth_type = v3password
+username = {{ .Values.global.nova_service_user | default "nova" }}{{ .Values.global.user_suffix }}
+password = {{ required ".Values.global.nova_service_password is missing" .Values.global.nova_service_password }}
+user_domain_name = {{.Values.global.keystone_service_domain | default "Default" }}
+region_name = {{.Values.global.region}}
```

Comment:
what's the reason for setting `region_name` here? it doesn't seem to be needed by https://docs.openstack.org/keystoneauth/ussuri/plugin-options.html#v3password .

Reply:
I copied the stuff over from another section. We have this setting in every section we configure auth in - in a lot of projects https://github.com/search?q=org%3Asapcc+region_name&type=code

https://github.com/openstack/keystoneauth/blob/112bcae1fbec355fcb58da07ae1d9f1adf8b77ba/keystoneauth1/loading/adapter.py#L105-L108 There it is. A little further up is defined what it's used for.